src/components/links/LinkStandalone.tsx

```diff
@@ -18,13 +18,43 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+import { css } from '@emotion/react';
 import styled from '@emotion/styled';
+import { LinkProps } from '.';
 import { LinkBaseStyled } from './LinkBaseStyled';
 
-export const LinkSubdued = styled(LinkBaseStyled)`
-  --color: 'currentColor';
-  --hover: var(--echoes-color-background-accent-default-hover);
-  --active: var(--echoes-color-background-accent-default-active);
+interface Props extends LinkProps {
+  iconLeft?: React.ReactNode;
```

Comment:
why not `leftIcon`?

Reply:
Not a particular reason, it was already named like that in figma, and in case we add a iconRight later both of them will be next to each other in autocompletion, which I think is nice.